src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -188,32 +180,32 @@ public Void call() throws Exception {
     @Override
     public String createDeviceMember(final String name, String ipAddress, String vserverIpAddress, String contactIpAddress,
             String gateway, String prefixLength) throws Exception {
-        DeviceEntity device = new DeviceEntity();
         DeviceMemberEntity deviceMember = new DeviceMemberEntity();
-        deviceMember.setParent(device);
-        device.setId(null);
+        updateMember(deviceMember, vserverIpAddress, contactIpAddress, ipAddress, gateway, prefixLength);
         deviceMember.setName(name);
         return createDeviceMember(deviceMember, null, this.vs.getId());
     }
 
-    public String createDeviceMember(DeviceMemberEntity entity, Long parentId, Long vsId) throws Exception {
+    public String createDeviceMember(DeviceMemberEntity entity, Long deviceId, Long vsId) throws Exception {
```

Comment:
create device member should just take vs.getMgrId() only.

Reply:
OK, I have eliminated vsId completely and using MgrId now as the OSC is using today.